ci-operator/config/rhobs/observability-operator/rhobs-observability-operator-main__periodic.yaml

```diff
@@ -0,0 +1,41 @@
+base_images:
+  osde2e:
+    name: osde2e
+    namespace: ci
+    tag: latest
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: e2e-test-harness
+  commands: |
+    export REPORT_DIR="$ARTIFACT_DIR"
+    export CONFIGS="rosa,stage,test-harness"
+    export ROSA_ENV="stage"
+    export ROSA_STS="true"
+    export ROSA_AWS_REGION="random"
+    export TEST_HARNESSES="quay.io/rhobs/observability-operator-test-harness:latest"
```

Comment:
@lihongyan1 Should we rebuild and push this image?

Reply:
I executed the following and get error, seems I have no permission to do the push
`make osd-e2e-test-push IMAGE_BASE="quay.io/rhobs/observability-operator"`